package.json

```diff
@@ -32,7 +32,7 @@
     "test": "./bin/run_test_server"
   },
   "dependencies": {
-    "alpaca": "voxmedia/alpaca#dev",
+    "alpaca": "caseymm/alpaca#dev-input-style",
```

Comment:
This will need to be switched before we can merge. I had to make a few tweaks to alpaca to get a few style related things in forms to work correctly and don't have access to the repo. I will submit a PR for this branch of alpaca as well.